packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -32,7 +32,8 @@ export default class TableResize implements EditorPlugin {
     constructor(
         private onShowHelperElement?: (
             elementData: CreateElementData,
-            helperType: 'CellResizer' | 'TableInserter' | 'TableResizer' | 'TableSelector'
+            helperType: 'CellResizer' | 'TableInserter' | 'TableResizer' | 'TableSelector',
+            tableOrTd: HTMLTableElement | HTMLTableCellElement
```

Comment:
Please make  this parameter optional to be compatible with older builds

Reply:
I might be wrong, but I think that TS will allow passing in functions that only take two arguments and the type check won't fail. But I am open to making the change also if I am missing something!

Reply:
hmm. ok. That makes sense to me.